src/utils/src/net/tap.rs

```diff
@@ -5,7 +5,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
 
-use crate::ioctl::{ioctl_with_mut_ref, ioctl_with_ref, ioctl_with_val};
```

Comment:
`utils` is defined as Rust edition 2018. In this edition it is preferred to use the keyword crate instead of the name of the current crate: https://doc.rust-lang.org/stable/edition-guide/rust-2018/module-system/path-clarity.html#the-crate-keyword-refers-to-the-current-crate

Reply:
Indeed but isn't the current crate name `utils`?

Reply:
This was a miss on my side, I thought that we are replacing `crate` with `utils`. Since `utils` re-exports the ioctls from vmm-sys-util, you can also refer to it as `crate::ioctl`. Feel free to disregard my comment.